sympy/printing/tests/test_repr.py

```diff
@@ -11,6 +11,7 @@
 from sympy.polys.agca.extensions import FiniteExtension
 
 x, y = symbols('x,y')
+Permutation.print_cyclic=True
```

Comment:
?

Reply:
This is the result of symbol (0,2,1) not equal to symbol (1,2) failure in the test_repl. I got the suggestion from the glitter channel to solve the issue